uqcsbot/advent.py

```diff
@@ -542,5 +546,13 @@ async def advent_winners(
 
 
 async def setup(bot: UQCSBot):
+    if os.environ.get("AOC_SESSION_ID") is not None:
+        SESSION_ID = os.environ.get("AOC_SESSION_ID")
```

Comment:
I'd rather this to happen within the `Advent` class imo, but I'm not toooo fussed as this cog is on the rewrite chopping block.